creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -607,7 +765,7 @@ class SnapbackSM {
     } finally {
       // Log decision tree
       try {
-        this.log(`processStateMachineOperation Decision Tree ${JSON.stringify(decisionTree)}`)
+        this.log(`processStateMachineOperation Decision Tree ${JSON.stringify(decisionTree, null, 2)}`)
```

Comment:
i think this may actually not be what we want as it would break up the logline into multiple lines

Reply:
reverting